infra/utils.py

```diff
@@ -41,21 +42,26 @@ def execute(command, location=None, check_result=False):
     check_result: Should an exception be thrown on failed command.
 
   Returns:
-    The stdout of the command, the error code.
+    stdout, stderr, error code.
 
   Raises:
     RuntimeError: running a command resulted in an error.
   """
 
   if not location:
     location = os.getcwd()
-  process = subprocess.Popen(command, stdout=subprocess.PIPE, cwd=location)
+  process = subprocess.Popen(command,
+                             stdout=subprocess.PIPE,
+                             stderr=subprocess.PIPE,
+                             cwd=location)
   out, err = process.communicate()
-  if check_result and (process.returncode or err):
-    raise RuntimeError('Error: %s\n Command: %s\n Return code: %s\n Out: %s' %
-                       (err, command, process.returncode, out))
-  if out is not None:
-    out = out.decode('ascii').rstrip()
+  out = out.decode('ascii')
+  err = err.decode('ascii')
+  if err:
+    logging.error('stderr of command \'%s\' is %s.', ' '.join(command), err)
```

Comment:
are you sure we need to log this every time?

Reply:
Switched error to debug.